content/go/concepts/interfaces/interfaces.md

```diff
@@ -0,0 +1,271 @@
+---
+Title: 'Interfaces'
+Description: 'An interface is a set of method signatures.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Control Flow'
+  - 'Interfaces'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/back-end-engineer-career-path'
+---
+
+<h2>Creating an Interface</h2>
```

Comment:
Can we move this? It should follow the introductory paragraph. Also, no need to use HTML formatting, we'll stick to markdown with the hashtags or pound symbol `##` (what I meant by h2)